libs/core/src/lib/multi-input/multi-input.component.ts

```diff
@@ -301,6 +329,8 @@ export class MultiInputComponent
             );
         }
 
+        this.footerLinkText = 'Show All (' + this.dropdownValues.length + ')';
```

Comment:
Using this approach users can't pass their own string with value in it. I would suggest the next approach

```
// Example of user's input
// [footerLinkText]="linkTextFn"
// linkTextFn = (valuesLength) => `String in different language ${valuesLength}`;
@Input()
footerLinkText: (valuesLength: number) => string;

// Use this field in the component's template
get _footerLinkText(): string {
  return this.footerLinkText(this.dropdownValues.length);
}
```